### PR TITLE
added tests for panInsideBounds

### LINKED ISSUE #8368: Implement test for 'panInsideBounds' method of Map

### PR DESCRIPTION
**Pull request by mikelowe5919**
Potential resolve #8368 
Please let me know how everything looks and if these are valid for the method.
This is my first time trying to contribute to an open source project.

### CODE REVIEW

File: spec/suites/map/MapSpec.js [hunk 1]
```diff
@@ -2016,6 +2016,37 @@ describe("Map", function () {
...
+			expect(map.panInsideBounds(bounds)).to.be(map);
+			expect(map.getCenter().distanceTo(expectedCenter)).to.be.lessThan(1);
+		});
+
+		it("pans to closest view in bounds", function () {
+			var bounds = L.latLngBounds([[41.8, -87.6], [40.7, -74]]);
+			var expectedCenter = L.latLng([41.59452223189, -74.2738647460]);
+			map.setView([50.5, 30.5], 10);
+			expect(map.panInsideBounds(bounds)).to.be(map);
+			expect(map.getCenter().distanceTo(expectedCenter)).to.be.lessThan(5);
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:2046]: Please use latlng validation for the center

File: spec/suites/map/MapSpec.js [hunk 2]
```diff
@@ -2016,6 +2016,37 @@ describe("Map", function () {
...
+				map.panInsideBounds(0, 0);
+			}).to.throwError();
+		});
+
+		it("doesn't pan if already in bounds", function () {
+			map.setView([0, 0]);
+			var bounds = L.latLngBounds([[-1, -1], [1, 1]]);
+			var expectedCenter = L.latLng([0, 0]);
+			expect(map.panInsideBounds(bounds)).to.be(map);
+			expect(map.getCenter().distanceTo(expectedCenter)).to.be.lessThan(1);
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:2038]: Please use latlng validation for the center
```suggestion
			expect(map.getCenter()).to.be.nearLatLng(expectedCenter);
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please checkout the requested changes

File: spec/suites/map/MapSpec.js (hunk 2 above)
Comment (mikelowe5919) [reply spec/suites/map/MapSpec.js]: Thank you for checking my pull request. I made the changes you requested. I hope this looks better. Please let me know if there is anything else I need to modify.

**Review by Falke-Design — APPROVED** [review]
Thank you for your contribution

**Pull request merged**